frontend/app/application.py

```diff
@@ -103,12 +103,26 @@ def set_markdown(software, fields):
             software[field] = None
 
 
+def fetch_team_member_data(team_member):
+    person_id = team_member['foreignKey']['id']
+    url = api_url + "/person/%s" % person_id
+    person_dictionary = requests.get(url).json()
+    if "error" in person_dictionary:
+        return team_member
+    team_member.update(person_dictionary)
+    return team_member
+ 
+
 @application.route('/software/<software_id>')
 def software_product_page_template(software_id):
     url = api_url + "/software_cache/%s" % software_id
     software_dictionary = requests.get(url).json()
     if "error" in software_dictionary:
         return flask.redirect("/", code=302)
+    for i in range(0,len(software_dictionary['related']['projects'])):
```

Comment:
Just a quick observation: 

Is there a reason why you're introducing a loop counter? If not, consider using something like

```suggestion
    for project in software_dictionary['related']['projects']:
```

and similarly,

```
for member in project['foreignKey']['team']
```

Reply:
The reason was the assignment inside the loop. I did not know that you could change a dictionary by changing the value of a a variable that was an alias for a part inside the dictionary. The code was updated and committed (e70c106). It is much more readable now. Thanks for the tip!